apis/kubedb/v1alpha1/mysql_types.go

```diff
@@ -74,6 +74,6 @@ type MySQLStatus struct {
 type MySQLList struct {
 	metav1.TypeMeta `json:",inline"`
 	metav1.ListMeta `json:"metadata,omitempty"`
-	// Items is a list of Xdb TPR objects
+	// Items is a list of MongoDB TPR objects
```

Comment:
Do not use pointer in List Items. See Postgres & Elasticsearch

Reply:
Fixed in #145